### PR TITLE
Update sandman_de.ts and change-log typo fixes

### PR DESCRIPTION
**Pull request by bastik-1001**
Fresh fork to initially translate a string that has been changed, while also removing its old source and fixing mistakes in the change-log, that codespell marked as bad.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,25 +12,25 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
 ### Added
-- added local template editorto the global settings window
+- added local template editor to the global settings window
 - added hidden message list
-- added option to prevent processes located outside teh sandbox from loading boxed dll's
--- to enable it use "ProtectHostImages=y"
-- add option to block box inter programs but not box extern once
+- added option to prevent processes located outside the sandbox from loading boxed DLLs
+-- to enable it, use "ProtectHostImages=y"
+- add option to block box inter programs but not box extern once (garbled sentence ???)
```
Comment (isaak654) [inline CHANGELOG.md:22]: This should be fixed as well.

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (isaak654) [reply CHANGELOG.md:22]: Line 22 should be fixed as well.

**Comment by bastik-1001** [comment]
I have no idea what this is supposed to mean, so I can't fix it. My intention was to fix the typos, so codespell does not complain anymore.

**Comment by bastik-1001** [comment]
I have an idea of what it could mean, but I am not sure.

"added option to block box intern programs but not box external ones"

If you think that is correct, I can change it.

**Comment by bastik-1001** [comment]
Please note that the change-log mentions 1.8.5, which has not been released and the latest pre-release is 1.9.0.

**Comment by isaak654** [comment]
> Please note that the change-log mentions 1.8.5, which has not been released and the latest pre-release is 1.9.0.

release-date-1
release-date-2

**Pull request merged**